docs/src/main/asciidoc/config-yaml.adoc

```diff
@@ -36,43 +61,115 @@ You can also just add the following dependency into your project:
 ----
 implementation("io.quarkus:quarkus-config-yaml")
 ----
+--
+. Create the `src/main/resources/application.yaml` file.
++
+TIP: You can use the `yml` or `yaml` file extension.
+
+. To avoid potential confusion, remove the `src/main/resources/application.properties` file.
++
+NOTE: If both files are present, the values of properties in `application.yaml` override any duplicate properties in `application.properties`.
+
+// meta:include::modules/proc_using-nested-object-configuration-with-yaml.adoc[leveloffset=+2]
+[id="proc_using-nested-object-configuration-with-yaml_{context}"]
+== Using nested object configuration with YAML
 
-Remove the `src/main/resources/application.properties` and create a `src/main/resources/application.yaml` file.
+In the YAML configuration file, you can define a nested element inside an existing element to set configuration properties for your {project-name} application.
```

Comment:
I don't understand this sentence and what it brings to the plate?

I mean, this is the basics of YAML so it's not an extra feature or something that should be in a separate paragraph?